src/gui/poses.js

```diff
@@ -147,34 +147,45 @@ exports.drawPoses = function(poses, _coords, _cameraPos) {
         return;
     }
 
-    const pointWidth = 960;
-    const pointHeight = 540;
+    // following naming is for landscape images (width is a longer side)
+    // image resolution associated with 2D point positions
+    const pointWidth = imageSize[0]; // 1920; // 960;
+    const pointHeight = imageSize[1]; // 1080; // 540;
+    let outWidth = 0, outHeight = 0;
+    let halfCanvasWidth = 0, halfCanvasHeight = 0;
     const portrait = gfx.width < gfx.height;
-    let outWidth = gfx.width;
-    let outHeight = gfx.width / pointWidth * pointHeight;
+    // iphones crop camera image along shorter side. Thus, longer side is taken to calulate scaling factor from camera image to display canvas (gfx.width/height) 
+    if (!portrait) {
+        outWidth = gfx.width;
+        outHeight = (outWidth / pointWidth) * pointHeight;
+        halfCanvasWidth = gfx.width / 2;
+        halfCanvasHeight = gfx.height / 2;
+    }
+    else {
+        outWidth = gfx.height;
+        outHeight = (outWidth / pointWidth) * pointHeight;
+        halfCanvasWidth = gfx.height / 2.0;
```

Comment:
nit: it's more standard to write 2 instead of 2.0, in js all numbers are (for most intents and purposes outside of very fancy optimizations) double-precision floating point

Reply:
done